recipes/ncexplorer/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "ncexplorer" %}
+{% set version = "0.7.2" %}
+{% set sha256 = "396665b81e52958a7cb508e91e84d3b0bd56c64fdbb0c6597929dbfb14ed2385"
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: 0.7.2.tar.gz
+  url: https://github.com/godfrey4000/ncexplorer/archive/0.7.2.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
```

Comment:
It looks like this package uses `numpy`, `xarray`, `matplotlib` and others. Please ensure that all run time dependencies are listed here. The actual dependencies are hard to pick up and test, because the  imports of various modules for this project no-op.